lib/screens/main/registerScreen/registerFirstPage.dart

```diff
@@ -0,0 +1,441 @@
+import 'dart:ui';
+import 'package:fitnessbudds/screens/main/registerScreen/registerSecondPage.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
+import 'package:flutter/services.dart';
+import 'package:fitnessbudds/models/user.dart';
+import 'package:image_picker/image_picker.dart';
+import 'dart:io';
+import 'package:hexcolor/hexcolor.dart';
+import 'package:flutter_cupertino_date_picker/flutter_cupertino_date_picker.dart';
+import 'package:fitnessbudds/screens/main/registerScreen/textFormField.dart';
+
```

Comment:
export all _buildWidgets to different files,
try to minimize your code. delete unneeded & unused lines